ORK1Kit/ORK1Kit/Common/ORK1TaskViewController.m

```diff
@@ -1491,7 +1538,7 @@ - (void)decodeRestorableStateWithCoder:(NSCoder *)coder {
         }
         
         if ([_task respondsToSelector:@selector(stepWithIdentifier:)]) {
-            _hasSetProgressLabel = [coder decodeBoolForKey:_ORK1HasSetProgressLabelRestoreKey];
+            _hasSetProgress = [coder decodeBoolForKey:_ORK1HasSetProgressRestoreKey];
```

Comment:
Changing the restoration key may break the progress UI logic when restoring a task across ResearchKit version changes.

Reply:
Good point, will revert.